package.json

```diff
@@ -27,8 +27,9 @@
     "storybook": "start-storybook -p 3001"
   },
   "dependencies": {
-    "builder-victory-component": "^2.1.3",
     "builder": "~2.9.1",
+    "builder-victory-component": "^2.1.3",
+    "d3": "^3.5.17",
```

Comment:
We are using d3 modules. Find the appropriate module rather than relying on all of d3.